bin/create-image.sh

```diff
@@ -36,26 +24,42 @@ create_snapshot(){
     openstack image set --name "$image_name" "$new_image_name"
 }
 
-create_security_group "$SECGROUP"
+create_keypair(){
+    local private_key=$1
+    local keypair_name=$2
 
-IMAGE_NAME=$(get_image_name "$DIST")
+    local tmpdir=$(mktemp -d)
+    local public_key=$tmpdir/$keypair_name
 
-INSTANCE_ID=$(launch_instance "$IMAGE_NAME")
+    openstack keypair delete $keypair_name
+    ssh-keygen -y -f $private_key > $public_key
```

Comment:
We might have a problem here. For running the autopkgtests we need the private key of the scalingstack keypair to be in the .ssh/config of the slaves. This private key is not the same as the ~/.ssh/id_rsa we are using, right?